src/main/java/gregtech/loaders/recipe/chemistry/PolymerRecipes.java

```diff
@@ -470,4 +469,37 @@ private static void polybenzimidazoleProcess() {
                 .output(dust, ChromiumTrioxide, 4)
                 .buildAndRegister();
     }
+
+    public static void polycaprolactamProcess() {
+        CHEMICAL_RECIPES.recipeBuilder().EUt(VA[HV]).duration(400)
+                .fluidInputs(Benzene.getFluid(1000), Hydrogen.getFluid(6000))
+                .fluidOutputs(Cyclohexane.getFluid(1000))
+                .notConsumable(new IntCircuitIngredient(1))
+                .buildAndRegister();
+
+        CHEMICAL_RECIPES.recipeBuilder().EUt(VA[LV]).duration(100)
+                .fluidInputs(Chlorine.getFluid(2000), NitricOxide.getFluid(2000))
+                .fluidOutputs(NitrosylChloride.getFluid(2000))
+                .buildAndRegister();
+
+        CHEMICAL_RECIPES.recipeBuilder().EUt(VA[MV]).duration(100)
+                .fluidInputs(Cyclohexane.getFluid(1000), NitrosylChloride.getFluid(1000))
+                .output(dust, Cyclohexanoxime, 19)
+                .fluidOutputs(HydrochloricAcid.getFluid(1000))
+                .buildAndRegister();
+
+        CHEMICAL_RECIPES.recipeBuilder().EUt(VA[HV]).duration(200)
```

Comment:
I'm not sure what this reaction is supposed to be. Cyclohexanone Oxime is rearranged into Caprolactam. This recipe should be `1000 Cyclohexanone Oxime + 1000 Sulfuric Acid -> 19 Caprolactam + 1000 Diluted Sulfuric Acid.`